### ISSUE TITLE
Need help... Geth (Ropsten)

### ISSUE THREAD

**Issue by amiDMast** [ethereum/go-ethereum#24976]
Hello, I updated geth to version 1.10.18 and now in the logs:
```
May 26 22:28:29 test-eth-node-01 geth[29340]: INFO [05-26|22:28:29.448] Legacy sync reached merge threshold      number=12,308,106 hash=a6fbe9..bdbf28 td=43,532,321,247,089,466 ttd=43,531,756,765,713,534
May 26 22:28:30 test-eth-node-01 geth[29340]: INFO [05-26|22:28:30.781] Legacy sync reached merge threshold      number=12,308,106 hash=a6fbe9..bdbf28 td=43,532,321,247,089,466 ttd=43,531,756,765,713,534
May 26 22:28:30 test-eth-node-01 geth[29340]: WARN [05-26|22:28:30.781] Local chain is post-merge, waiting for beacon client sync switch-over...
May 26 22:28:39 test-eth-node-01 geth[29340]: INFO [05-26|22:28:39.821] Legacy sync reached merge threshold      number=12,308,106 hash=a6fbe9..bdbf28 td=43,532,321,247,089,466 ttd=43,531,756,765,713,534
May 26 22:28:42 test-eth-node-01 geth[29340]: INFO [05-26|22:28:42.506] Legacy sync reached merge threshold      number=12,308,106 hash=a6fbe9..bdbf28 td=43,532,321,247,089,466 ttd=43,531,756,765,713,534
May 26 22:28:42 test-eth-node-01 geth[29340]: WARN [05-26|22:28:42.507] Local chain is post-merge, waiting for beacon client sync switch-over...

```
And sync (Ropsten) stoped on 12308105 block.

Please tell me what is needed to continue the synchronization?

**Comment by Zachinquarantine** [comment]
See [this tweet](https://twitter.com/vdWijden/status/1529895417881800714) for info on how to get a Ropsten client to sync.
TL;DR: you can use `--override.terminaltotaldifficulty 100000000000000000000000` to allow your client to start to sync again and you can run `debug.setHead("0xBBCE88")` in [the console](https://geth.ethereum.org/docs/interface/javascript-console) to sync the right blockchain.

**Comment by karalabe** [comment]
Here's a small post mortem about the issue. https://notes.ethereum.org/@timbeiko/ropsten-ttd-override
